index.html

```diff
@@ -711,6 +711,58 @@ <h3>Hashlinks</h3>
 
     </section>
 
+    <section>
+      <h2>Progressive Trust</h2>
+      <em>This section is non-normative.</em>
+
+      <p>
+Entities that use <a>verifiable credentials</a> and
+<a>verifiable presentations</a> should follow protocols that enable progressive
+trust.
+      </p>
+
+      <section>
+        <h3>Data Minimization</h3>
+        <p>
+Data minimization is a principle that encourages <a>verifiers</a> to request the
+minimum amount of data necessary from <a>holders</a>, and for <a>holders</a> to
+only provide the minimum amount of data to <a>verifiers</a>.
+        </p>
+      </section>
+
+      <section>
+        <h3>Selective Disclosure</h3>
+
+        <p>
+Selective disclosure is the ability of a <a>holder</a> to select which elements
+from a <a>verifiable credential</a> to share with a <a>verifier</a>. There are
+a couple of different methods which support selective disclosure:
+        </p>
+        <ul>
+          <li>
+<b>Atomic Credentials</b> - These are <a>verifiable credentials</a> which
+consist of a single claim. An <a>issuer</a> may provide a set of atomic
+credentials that duplicates the claims of a standard credential.
+          </li>
+          <li>
+<b>Selective Disclosure Signatures</b> - Certain signature schemes natively
+support selective disclosure of <a>verifiable credential</a> elements. One
+example of these is Camenisch-Lysyanskaya signatures.
```

Comment:
Would love a reference link for Camenisch-Lysyanskaya sigs. In fact, each section could use a reference.

Reply:
added references to CL-sigs and other docs in (0e91eea)[https://github.com/w3c/vc-imp-guide/pull/14/commits/0e91eeafe33a5a1ed8ba2dea4c26dbfb5f6d8367]